.github/workflows/CI.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
As we are building on top of Cabana master, we might want to add a cron build here as well. Once a week?

Reply:
Sounds good to me

Reply:
Done.